layouts/partials/comments/isso.html

```diff
@@ -0,0 +1,6 @@
+{{- $issoURL := .Page.Site.Params.comments.isso.url -}}
+
+<script data-isso="{{ $issoURL }}"
+        {{- with .Page.Site.Params.comments.isso.extra }} {{ . | safeHTMLAttr }} {{ end -}}
+        src="{{ $issoURL }}js/embed.min.js"></script>
```

Comment:
Is there any guaranty that the script file is always gonna be in that path?

Reply:
Yep. It's either this or `embed.dev.js` for the bundled but unminified version.

Reply:
(One *could* use a CDN to host the JS, but since Isso is mostly used by privacy-concious folks, I doubt it makes a lot of sense... And since the JS and the backend must have the same version, I don't see any good reason to use something else.)

Reply:
Or maybe someone, for some reason, can host it in a different directory, who knows. Let's not hope for the best. Use `.Page.Site.Params.comments.isso.scriptSrc` for that.